build.sbt

```diff
@@ -13,6 +13,10 @@ val scalaTestVersion = "3.1.1"
 
 lazy val `monster-xml-to-json-list` = project
   .in(file("."))
+  .aggregate(`xml-to-json-list-clp`, `xml-to-json-list-template`)
+
+lazy val `xml-to-json-list-clp` = project
+  .in(file("clp"))
```

Comment:
Making the build multi-project was the simplest way to enable using the new plugin. The downside is I had to move all the existing code into a sub-directory.